packages/file_selector/file_selector/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.8.3+1
```

Comment:
Add a blank line between the heading and the items, for consistency with the rest of the file.